media_cdn/dualtoken.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+#
+# Copyright 2022 Google, Inc.
```

Comment:
It's 2023. Don't live in the past!

Reply:
Hi @m-strzelczyk, unfortunately I did not notice this earlier. I make a note of this in my next PR. Thanks